cds_ils/importer/providers/cds/rules/journal.py

```diff
@@ -133,22 +131,26 @@ def access_urls(self, key, value):
     access_type_mapped = []
     val_x = clean_val("x", value, str)
     access_type_list = list(map(int, val_x if val_x else []))
+
     for i in access_type_list:
         access_type = mapping(ACCESS_TYPE, str(i), raise_exception=True)
         access_type_mapped.append(access_type)
+
+    sub_3 = clean_val("3", value, str)
+    sub_z = clean_val("z", value, str)
+    electronic_volumes_description = f"{sub_3} ({sub_z})"
```

Comment:
super minor
```suggestion
    electronic_volumes_description = f"{sub_3} ({sub_z})".trim()
```
or strip()

so if one is empty, there is no extra space